src/main/java/lotto/record/ProfitRate.java

```diff
@@ -0,0 +1,31 @@
+package lotto.record;
+
+import static lotto.enumerate.ConfigInteger.DECIMAL_PLACE;
+import static lotto.enumerate.ConfigInteger.LONG_ROUND_DIVIDE_NUMBER;
+import static lotto.enumerate.ConfigInteger.LONG_ROUND_NUMBER;
+import static lotto.enumerate.ConfigString.STRING_FORMAT;
+import static lotto.enumerate.ConfigString.STRING_INDEX;
+import static lotto.enumerate.ErrorCode.PROFIT_RATE_DECIMAL_NOT_ONE;
+import static lotto.enumerate.ErrorCode.PROFIT_RATE_NOT_ROUNDED;
+import static lotto.enumerate.Message.PROFIT_RATE_IS;
+
+import lotto.util.ExceptionCodeThrow;
+
+public record ProfitRate(double originalRate, double roundedRate) {
+
+    public ProfitRate {
+        if ((double) Math.round(originalRate * LONG_ROUND_NUMBER.getInt()) / LONG_ROUND_NUMBER.getInt()
+                * LONG_ROUND_DIVIDE_NUMBER.getInt() != roundedRate) {
```

Comment:
동의합니다. 메서드를 활용하면 조건문이 확인하고 싶은 내용이 어떤건지 훨씬 간편하게 파악할 수 있을 거에요
```suggestion
        if (isProfitRateRounded(originalRate, roundedRate)) {
```